src/msw/bitmap.cpp

```diff
@@ -176,10 +176,9 @@ wxIMPLEMENT_DYNAMIC_CLASS(wxBitmapHandler, wxObject);
         //  (a) if hdc is specified, the caller explicitly wants DDB
         //  (b) otherwise, create a DIB if depth >= 24 (we don't support 16bpp
         //      or less DIBs anyhow)
-        //  (c) finally, create DIBs under Win9x even if the depth hasn't been
+        //  (c) finally, create DIBs under MSW even if the depth hasn't been
```

Comment:
This "MSW" doesn't really make sense here, all this code is MSW-specific and used only under it.